vm_test.go

```diff
@@ -149,9 +149,9 @@ func TestHyperRelocationNewcontainer(t *testing.T) {
 	err = vm.relocateHyperCommand(cmd)
 	assert.NotNil(t, err)
 
-	vm.Close()
```

Comment:
In real conditions, qemu can exit while we are connected to it and we can close the sockets before qemu exits. Is that a problem?